src/.nuspec/Uno.Resizetizer.targets

```
@@ -672,4 +681,20 @@
 		<RemoveDir Directories="$(_ResizetizerIntermediateOutputRoot)" Condition="Exists ('$(_ResizetizerIntermediateOutputRoot)' )" />
 	</Target>
 
+	<Target Name="_GenerateWindowUnoIconExtension"
+			BeforeTargets="Build;CoreCompile"
+			Inputs="@(UnoIcon)"
+			Outputs="@(_WindowIconExtension)">
+		<WindowIconGeneratorTask
```

need to rename here as well.

```suggestion
		<WindowIconGeneratorTask_v0
```